indigo_app/static/javascript/indigo.js

```diff
@@ -72,10 +72,10 @@ function bootstrapIndigo(exports) {
   $.fn.datepicker.defaults.autoclose = true;
 
   // tooltips
-  $('[title]:not(.notooltip)').tooltip({
-    container: 'body',
-    placement: 'auto',
-  });
+  // $('[title]:not(.notooltip)').tooltip({
+  //   container: 'body',
+  //   placement: 'auto',
+  // });
```

Comment:
This bit is now throwing errors that cause the taxonomy toc component not to load. I commented it out because I was not sure what it was doing. I made an update to the timestamps.js tooltips that is working.

Reply:
![image](https://github.com/laws-africa/indigo/assets/25079238/3f524d5e-522f-4a0d-8006-f7622b8f5a1b)

Reply:
This initialises tooltips on elements with a `title` attribute.

Can you pull your new tooltip init code from relativeTimestamps in here instead, and ensure it works for `data-bs-tooltip` as well as elements with `title` attribute?

Reply:
done :+1: